server.js

```diff
@@ -139,7 +139,9 @@ client.on('message', message => {
   }
   var triggerlist = Object.keys(triggers);
   for (var i = 0; i<triggerlist.length; i++){
-    if(message.content.toLowerCase().includes(triggerlist[i])){
+    var pattern = new RegExp("[^a-z]" + trigger + "[^a-z]", "i");
```

Comment:
Can we move the RegExp creation out of the on message function so it only occurs when Digger starts? Then we just match against a list of known regular expressions. It's a bigger refactor but should be more performant

Reply:
Also, is the goal of the regex to find word boundaries (start of string, space, etc)? We can use `\b` for that instead of `[^A-Z]`.